test/unit/helpers/mocks/Util.js

```diff
@@ -91,18 +91,22 @@ function ($, _, Backbone, Q, Duo) {
       }
 
       var deferred = $.Deferred();
-      if (xhr.status > 0 && xhr.status < 300) {
-        // $.ajax send (data, textStatus, jqXHR) on success
+
+      (function (textOnly) {
```

Comment:
The function inside the `setTimeout` is called async after the current.
Without the closure function, `textOnly` will always be false.